kroxylicious-operator/src/main/resources/META-INF/fabric8/virtualkafkaclusters.kroxylicious.io-v1.yml

```diff
@@ -79,6 +79,17 @@ spec:
                           maxLength: 253
                           minLength: 1
                           type: string
+                ingressRefs:
```

Comment:
`description`

Reply:
pushed up descriptions and CEL validation for restricting group to `kroxylicious.io` and kind to `KafkaProxy`.